GroupProject/Pages/Index.cshtml

```diff
@@ -7,32 +7,28 @@
 
 
 <div class="text-center">
-    <h1 class="display-4">Welcome</h1>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
-</div>
-
-<div class="text-center">
-    <hi class="display-4"> Player Info</hi>
-    <hr />
+    <h4 class="display-4"> Player Information</h4>
+    <hr>
     <table border="1" width="90%">
         <tr><th>Player Name</th><th>Team Name</th><th>Data Joining Team</th></tr>
         @{
             foreach (QuickType.PlayerInfo playerInfo in playerInfos)
             {
+                string date = playerInfo.StartDate.ToString();
+                string[] dars = date.Split('/'); 
+                string day = dars[0]; 
+                string month = dars[1]; 
+                string year = dars[2]; 
+                string[] year1 = year.Split(' '); 
+                string yearclean = year1[0]; 
+
```

Comment:
It may be worth doing the split logic in the codebehind, and then expose "day, month, year" as attributes of palyerInfo.